plugins/azure/resoto_plugin_azure/resource/network.py

```diff
@@ -3844,8 +3809,8 @@ def connect_in_graph(self, builder: GraphBuilder, source: Json) -> None:
             path="/subscriptions/{subscriptionId}/providers/Microsoft.Network/virtualNetworks",
             path_parameters=["subscriptionId"],
             query_parameters=["api-version"],
-            compared_property=lambda r: r["properties"]["addressSpace"]["addressPrefixes"],
-            binding_property=lambda r: r["id"],
+            first_property=lambda r: r["properties"]["addressSpace"]["addressPrefixes"],
+            second_property=lambda r: r["id"],
         )
```

Comment:
This call is not required - it fetches all virtual networks, which have been already fetched and are available in the graph.